apps/api/src/auth/services/auth.interceptor.ts

```diff
@@ -43,4 +50,12 @@ export class AuthInterceptor implements HonoInterceptor {
       return await next();
     };
   }
+
+  private async getValidUserId(bearer: string, c: Context) {
```

Comment:
I'd put this method into `AuthTokenService` but I don't want that one to depend on http context. And seems like this method needs that in any way. Any thoughts @Redm4x ?